### ISSUE TITLE
Add support for Windows

### ISSUE THREAD

**Issue by mardukbp** [version-fox/vfox-elixir#8]
vfox is a cross-platform SDK manager. Therefore, it is expected that the plugins also work on all platforms.

Moreover, it would be great to be able to install Elixir (and Erlang) without a graphical installer, since the graphical installer for Windows requires admin rights. Since most people at universities and enterprises have no admin rights, they cannot use Elixir.

**Comment by yeshan333** [comment]
I have already do some work in this [pull request](https://github.com/version-fox/vfox-elixir/pull/4), but still need to test it. There are still some problems at present.

**Comment by mardukbp** [comment]
Awesome! I saw that you install Erlang and Elixir by executing the .exe installers. Like I said they require admin rights. I already opened an issue on the Erlang OTP repo hoping that they will create an installer that doesn't need admin rights. AFAIK it requires admin rights because it executes the installer for Microsoft Visual C++ Redistributable in order to install vcruntime140.dll. But this DLL can be shipped with the installer, like Python does. It is present in the Windows 2022 GH image.

**Comment by yeshan333** [comment]
I have released a new plugin version [v0.4.0](https://github.com/version-fox/vfox-elixir/releases/tag/v0.4.0) to support this. You can have a try~. @mardukbp 

Note: 
- **vfox version require v0.5.3+**
- update vfox-elixir plugin

**Comment by mardukbp** [comment]
It works! But, I noticed that the elixir-plugin does not know that Elixir depends on Erlang. Therefore one has to install Erlang by hand. Does vfox support dependencies on other plugins?

**Comment by yeshan333** [comment]
Yes, you must clear to know whether the Erlang version that the Elixir version you want to install depends on is already installed. Best used in combination with the [vfox-erlang ](https://github.com/version-fox/vfox-erlang) plugin.

Maybe, it will not be a problem to use it with asdf-erlang, but I have not tested this scenario.

Vfox should not support interdependence between plug-ins at present.

**Comment by mardukbp** [comment]
Yeah, with vfox-erlang it's a breeze. I like that vfox is fast and intuitive. Thank you for all your efforts!
